DESCRIPTION

```diff
@@ -9,7 +9,6 @@ Depends:
     ccao,
     conflicted,
     dplyr,
-    embed,
```

Comment:
The embed package has super heavy dependencies (keras, tensorflow), so getting rid of it lightens the dependencies of the main lockfile quite a bit.